src/test/rpc/Submit_test.cpp

```diff
@@ -185,8 +185,9 @@ class Submit_test : public beast::unit_test::suite
             {
                 return;
             }
-            BEAST_EXPECT(client.reply.engine_result().result() == "tefALREADY");
-            BEAST_EXPECT(client.reply.engine_result_code() == -198);
+            BEAST_EXPECT(
+                client.reply.engine_result().result() == "tefPAST_SEQ");
+            BEAST_EXPECT(client.reply.engine_result_code() == -190);
```

Comment:
I was not aware of that, but you are correct.  I believe that's because `tefALREADY` is no longer accessible by a normal transaction.  The sequence check happens first in `invoke_preclaim` which returns `tefPAST_SEQUENCE` before calling `checkPriorTxAndLastLedger()`.  Any transaction that meets the requirements for `tefALREADY` also meets the requirements for `tefPAST_SEQUENCE`.  And the sequence check is cheaper to perform, so it makes sense to do that first.

It looks like we could gin up a unit test to exercise `tefALREADY` using pseudo transactions.  But that would be in a different test file.

In effect I don't think we've actually lost any test coverage here, but we have identified some (currently) inaccessible code.

Reply:
:thinking: Yeah, that makes sense. I'm not too worried about it. (context: `tefALREADY`)